src/_vendor/assets/stylesheets/uswds/elements/_inputs.scss

```diff
@@ -9,7 +9,7 @@ $input-padding-vertical: 1rem;
 $input-height-exact: (
   ($base-font-size * $input-line-height) +
   ($input-padding-vertical * 2) +
-  (strip-units($input-border-width) * 2rem / $base-font-size)
+  (rem($input-border-width * 2))
```

Comment:
Reverting to the original and will try and solve the deprecation warning down the line…